tests/app/dao/test_templates_dao.py

```diff
@@ -96,6 +96,28 @@ def test_update_template(sample_service, sample_user):
     assert dao_get_all_templates_for_service(sample_service.id)[0].name == 'new name'
 
 
+def test_update_template_with_reply_to_email(sample_service, sample_user):
+    data = {
+        'name': 'Sample Template',
+        'template_type': "sms",
+        'content': "Template content",
+        'service': sample_service,
+        'created_by': sample_user,
+        'reply_to_email': "some@one.com"
+    }
+    template = Template(**data)
+    dao_create_template(template)
+    created = dao_get_all_templates_for_service(sample_service.id)[0]
+    assert created.name == 'Sample Template'
+    assert created.reply_to_email == "some@one.com"
+
+    created.name = 'new name'
+    created.reply_to_email = "testing@change.com"
+    dao_update_template(created)
```

Comment:
This tests the function that the API endpoint code calls, but it doesn't test the API endpoint code, contrary to your test name.  You should test both.  You also haven't tested that the new TemplateHistory row gets created as expected.

Reply:
I'm pretty sure this test (which I did base off one of the other tests) was not to test the API endpoint but being inside the `tests/app/dao` section was to only test the function that I created to update the template. I will see if I can locate the API tests and update them. I am also in the process of consolidating all the `update template` methods into a single one per your suggestion so these will have to change to accomidate that test of the DOA method updates anyways.

Reply:
Test ended up not really being needed and has been removed.